Ports/ssmtp/package.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env -S bash ../.port_include.sh
+port=ssmtp
+version=2.64-11
+files="https://salsa.debian.org/debian/ssmtp/-/archive/debian/${version}/ssmtp-debian-${version}.tar.gz ssmtp-debian-${version}.tar.gz 82abe0fb54e2ee209e9f8745498ce5f4e35f2f2b1bf95bd5e6dfbcfc61a8ebe2"
+auth_type='sha256'
+useconfigure=true
+workdir="ssmtp-debian-${version}"
+configopts=("--enable-ssl" "--enable-md5auth" "--prefix=${SERENITY_INSTALL_ROOT}/usr/local")
```

Comment:
Uninterpolated strings should use single quotes. Also putting these options on separate lines will make diffs easier to read in the future.